lib/CMakeLists.txt

```diff
@@ -6,7 +6,11 @@ if (HCC_RUNTIME_DEBUG)
   add_definitions(-DKALMAR_DEBUG=1)
 endif (HCC_RUNTIME_DEBUG)
 
-include_directories(${CMAKE_CURRENT_BINARY_DIR} ${LIBCXX_HEADER} )
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+
+if (USE_LIBCXX)
+  include_directories(${LIBCXX_HEADER})
```

Comment:
@scchan I've merged PR #206 from @pfultz2 , could you rebase and modify this PR according to what Paul recommends?